src/learning/infer.py

```diff
@@ -0,0 +1,31 @@
+import torch
+import torchvision
+from torchvision.utils import save_image
+import utils
+from modules import Generator
+
+MODEL_PATH = 'gpu_model'
+VALIDATION_DATA_PATH = 'test_A'
+
+DISPLAY_IMAGE = False
+N_VALIDATION_IMAGES = 3
+N_RESIDUAL_BLOCKS = 9
+USE_DROPOUT = False
+IMAGE_SIZE = (256, 256)
+
+
+def main():
+    model = Generator(N_RESIDUAL_BLOCKS, USE_DROPOUT)
+    device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+    model.load_state_dict(torch.load(MODEL_PATH, map_location=device))
+    transform = utils.get_transform(IMAGE_SIZE)
+    validation_data = torchvision.datasets.ImageFolder(
+        root=VALIDATION_DATA_PATH, transform=transform)
```

Comment:
Maybe I'm missing something, but it's crucial to set the model to evaluation mode. Doing so turns off Dropout and affects the behaviour of BatchNorm, for instance: https://pytorch.org/docs/master/nn.html#torch.nn.BatchNorm1d, https://pytorch.org/docs/stable/nn.html?highlight=eval#torch.nn.Module.eval, https://discuss.pytorch.org/t/what-does-model-eval-do-for-batchnorm-layer/7146
```suggestion
        root=VALIDATION_DATA_PATH, transform=transform)
        model.eval()
```

Reply:
Awesome!